demo/src/main/scala/com/tresata/spark/scalding/demo/DemoJobSequenceClass.scala

```diff
@@ -0,0 +1,23 @@
+package com.tresata.spark.scalding.demo
```

Comment:
I saw you added them and that's why I did too. Could be useful to people to test things with a simple example first I guess. Let's leave it.